### ISSUE TITLE
Ship with systemd

### ISSUE THREAD

**Issue by sonhanguyen** [gitpod-io/openvscode-server#160]
[`code-server` ships a `systemd` unit](https://github.com/cdr/code-server/pull/1997/commits/255fa37e1d058cce917eb75c2da20c2c806a52c2) for more convenient installation.

 I know this project wants to minimize deviation from official vscode, just wondering if this could be considered as the chance of conflict is low

I can also offer myself to do the work

**Comment by andymac4182** [comment]
This could be a Guide https://github.com/gitpod-io/openvscode-server/tree/docs/guides would be fairly easy to get up and running in there.

**Comment by akosyakov** [comment]
I don't think it should go in this repo. We build docker images though here: https://github.com/gitpod-io/vscode-releases  Guide is fine as well. If someone wants to contribute go ahead.

**Comment by nobiit** [comment]
Hey @akosyakov 
Adding systemd to a docker image is quite simple. But will you allow running that docker image with privileged mode and mount cgroup ?
Is it possible to add custom flags for `docker run` in gitpod.yml ?
